src/server/api/controllers/channel-members.controller.js

```diff
@@ -2,6 +2,20 @@ const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
 const moment = require('moment-timezone');
 
+const getChannelMembersByChannelId = async (channelId) => {
```

Comment:
Why not add missing fields to `getChannelMemberById` below which does the same thing? We shouldn't have too many functions doing the same thing.